website/docs/plugin/framework/migrating/attributes-blocks/blocks.mdx

```diff
@@ -14,6 +14,17 @@ This page explains how to migrate nested blocks that are not computed (i.e., do
 [Blocks with Computed Fields](/plugin/framework/migrating/attributes-blocks/blocks-computed) for more details
 about migrating nested blocks that contain fields that are computed.
 
+Refer to [Attribute Fields](/plugin/framework/migrating/attributes-blocks/fields) for details regarding the relationship
```

Comment:
Since there are a variety of differences between blocks and attributes, I think this page might warrant its own table. The prior SDK allowed developers to do unsupported things, like declare `Sensitive` on blocks (https://github.com/hashicorp/terraform-plugin-sdk/pull/819) and we already have special callouts in paragraph form about `Computed` and `Required`. Therefore I think we should walk through the various fields and how they relate to blocks:

| SDK Schema Field | Framework |
|---|---|
| Computed | Link to computed block page |
| Optional | N/A - no implementation required |
| Required | `listvalidator.IsRequired()` and `setvalidator.IsRequired()` (let's omit any discussion on single nested blocks, they weren't possible to declare before) |
| Sensitive | N/A - only supported on attributes |

etc. 😄

Reply:
Have added a table into the Blocks page.